crates/aptos/CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ All notable changes to the Aptos CLI will be captured in this file. This project
 # Unreleased
 
 - Add `--node-api-key` flag to `aptos move replay` to allow for querying the fullnode with an API key.
-
+- Add `--chunk-size` flag to allow configuring chunk size for chunked publish mode.
```

Comment:
You should also mention that you're lowering the default value from x to y.

Reply:
Fixed!